hugegraph-llm/src/tests/operators/llm_op/test_disambiguate_data.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from hugegraph_llm.llms.init_llm import LLMs
+from hugegraph_llm.operators.llm_op.disambiguate_data import DisambiguateData
+
+
+class TestDisambiguateData(unittest.TestCase):
+    def setUp(self):
+        self.triples = {
+            "triples": [
+                (' "Alice "', ' "Age "', ' "25 "'),
```

Comment:
Why do we need extra quotes and spaces? Can't `('Alice', 'Age', '25')` represent a SPO?

Reply:
Of course, these triples are output from ernie-bot, maybe more like real scenes?